package.json

```diff
@@ -7,5 +7,5 @@
     "@commitlint/config-conventional": "^19.8.0",
     "husky": "^9.1.7"
   },
-  "packageManager": "pnpm@9.11.0+sha512.0a203ffaed5a3f63242cd064c8fb5892366c103e328079318f78062f24ea8c9d50bc6a47aa3567cabefd824d170e78fa2745ed1f16b132e16436146b7688f19b"
+  "packageManager": "pnpm@10.18.3"
```

Comment:
Not sure about this one - we have a fixed pnpm version in our dockerfiles because there was a bug a while back with a newer version so we might just want to keep the old one as is...